images/airflow-managed-services/Dockerfile

```
@@ -11,12 +11,16 @@ RUN mv $GOPATH/bin/pkger /usr/local/bin/
 ENV GOFLAGS=
 RUN apt update -y
 RUN apt upgrade -y
-RUN apt install jq unzip bsdmainutils -y
+RUN apt install jq unzip bsdmainutils build-essential libncurses5-dev zlib1g-dev libnss3-dev libgdbm-dev libssl-dev libsqlite3-dev libffi-dev libreadline-dev curl libbz2-dev -y
 RUN curl -L $(curl -s https://api.github.com/repos/openshift/rosa/releases/latest | jq -r ".assets[] | select(.name == \"rosa-linux-amd64\") | .browser_download_url") --output /usr/local/bin/rosa
 RUN curl -L $(curl -s https://api.github.com/repos/openshift-online/ocm-cli/releases/latest | jq -r ".assets[] | select(.name == \"ocm-linux-amd64\") | .browser_download_url") --output /usr/local/bin/ocm
 RUN chmod +x /usr/local/bin/rosa && chmod +x /usr/local/bin/ocm
 RUN /usr/local/bin/rosa download openshift-client
 RUN tar xzvf openshift-client-linux.tar.gz
+RUN curl "https://www.python.org/ftp/python/3.9.9/Python-3.9.9.tar.xz" -o "python39.tar.xz"
+RUN tar -xvf "python39.tar.xz"
```

This statement can be optimized with:

```suggestion
RUN curl https://www.python.org/ftp/python/3.9.9/Python-3.9.9.tar.xz | tar xf
```

To prevent creating an extra layer